.circleci/config.yml

```diff
@@ -1,4 +1,12 @@
 aliases:
+  - &drupal_defaults
+    docker:
+      - image: andrewberry/drupal_tests:0.3.4
+      - image: selenium/standalone-chrome-debug:3.7.1-beryllium
+      - image: mariadb:10.3
```

Comment:
why are we using mysql and not sqlite?

Reply:
@davidhwang This is just copied directly from @deviantintegral :)

Reply:
Trying for the least friction to get testing working :)

Reply:
sure, no worries

Reply:
So we actually started with sqlite, and ran into a few issues and [switched to mariadb](https://github.com/deviantintegral/drupal_tests/pull/5).

1. Behat tests would sporadically fail if there were enough concurrent requests with sqlite lock exceptions.
1. There's a few contributed modules out there (we ran into a module depending on dynamic_entity_reference) that depended on mariadb-only features like triggers. With a mariadb container basically being free under circleci we've yet to see any issues since switching.

Reply:
One nice reason to use sqlite would be that we tell people to test out drupal-admin-ui using sqlite too. This way we would be more in line with the actual environment people use.